### PR TITLE
Add Sujit

### PR DESCRIPTION
**Pull request by sujitmohanty**
Added sujit as a member

### CODE REVIEW

File: ids/mapping.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
   "swaraj": "swarajpure",
   "sweta": "karnsweta92",
   "taksh": "takshch",
-  "tejas": "tejasberi"
+  "tejas": "tejasberi",
+  "sujit": "sujitmohanty"
```
Comment (ankushdharkar) [inline ids/mapping.json:18]: Alphabetic order not maintained

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
- Alphabetic order not maintained
- Picture is not square
- Face does not cover > 60% of the image

**Review by sujitmohanty — COMMENTED** [review]
-Fixed order.
-Cropped Image to ensure it's a square and meets min. size requirements

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
Why were 16 files changed? Please only push the code that pertains to your entry

**Review by sujitmohanty — COMMENTED** [review]
-Undid Prettier auto-formatting
-Re-fixed alphabetical order
-Changed Image

**Review by swarajpure — APPROVED** [review]
Looks good to me.

**Pull request merged**